examples/test_sync_example.py

```diff
@@ -0,0 +1,21 @@
+"""Testing Syncrhonous Extraction"""
+
+import sys
+
+from any_parser import AnyParser
+
+sys.path.append(".")
+sys.path.append("..")
+sys.path.append("../..")
+
+# define API key and file path
+API_KEY = "U1xoe4F7qQ9MGJkiMFtR240r3r2iF6NQaAMUEYYc"
+FILE_PATH = "./sample_data/stoxx_index_guide_0003.pdf"
+
+# create instance of AnyParser
+ap = AnyParser(API_KEY)
+
+# synchronous extract job (both extraction and result fetching, 60s timeout)
+md_output, total_time = ap.extract(FILE_PATH)
+print(md_output)
+print(total_time)
```

Comment:
let's move this to the proper test folder like what we did for a separate project https://github.com/CambioML/uniflow-llm-based-pdf-extraction-text-cleaning-data-clustering/tree/main/tests.

Also, you should immediate disable this API key because this is a public repo and anybody can see this.